os/fs/smartfs/smartfs.h

```diff
@@ -249,6 +249,8 @@
 #undef  CONFIG_SMARTFS_DYNAMIC_HEADER
 #endif
 
+#define AVAIL_BYTES(v) (v)->fs_llformat.availbytes	/* Bytes in the sector excluding MTD header */
```

Comment:
maybe can use full **AVAILABLE_BYTES**, but this is also ok.

Reply:
Aim is to avoid writing the excessively long variable name.
So using an alias as short as possible seems better?